src/main/java/uk/gov/companieshouse/ocr/api/image/extracttext/ImageOcrApiController.java

```diff
@@ -4,6 +4,7 @@
 import java.util.concurrent.CompletionException;
 
 import org.apache.commons.lang.time.StopWatch;
+import org.codehaus.plexus.util.StringUtils;
```

Comment:
Hi Steve, do we need the dependency org.codehaus.plexus.util.StringUtils for one test? Is it a dependency we need as part of other dependencies like Spring? If not I'd avoid using it because the more dependencies we have the harder it is to manage.

Reply:
Updated to use Apache Commons